File: Latihan A/cpp/Processor.cpp
// Library
#include <iostream>
#include <string>
using namespace std;
// deklarasi kelas
class Processor{
    // variabel private
    private:
        string name;
        int price;
    // variabel dan method publik
    public:
        // konstruktor
        Processor(){

        }
        Processor(string name, int price){
            this->name = name;
            this->price = price;
        }
        // get dan set
        void setname(string name){
            this->name = name;
        }
        string getname(){
            return this->name;
        }
        void setprice(int price){
            this->price = price;
        }
        int getprice(){
            return this->price;
        }
        // destruktor
        ~Processor(){}
};

File: Latihan B/php/Processor.php
<?php
// deklarasi kelas
class Processor{
	//variabel lokal
	private $name;
    private $price;
	//konstruktor
	public function __construct($name = "", $price = 0){
		$this->price = $price;
        $this->name = $name;
	}
	//get dan set setiap variabel
	public function setprice($price){
		$this->price = $price;
	}
	public function getprice(){
		return $this->price;
	}
	public function setname($name){
		$this->name = $name;
	}
	public function getname(){
		return $this->name;
	}
	// destruktor
	public function __destruct(){
	}
}
?>